packages/harvester/harvester-csi-driver/generated-changes/patch/Chart.yaml.patch

```diff
@@ -10,10 +10,10 @@
    catalog.cattle.io/os: linux
    catalog.cattle.io/permits-os: linux
 -  catalog.cattle.io/rancher-version: '>= 2.6.1-0 < 2.8.0-0'
-+  catalog.cattle.io/rancher-version: '>= 2.8.0-0 < 2.9.0-0'
++  catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.10.0-0'
    catalog.cattle.io/release-name: harvester-csi-driver
    catalog.cattle.io/ui-component: harvester-csi-driver
```

Comment:
the `Chart.yaml.patch` needs to be re-generated.
original is:
```
  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.26.0-0'
```
new:`'>= 1.23.0-0 < 1.30.0-0'`

Reply:
Hi @w13915984028, Thanks for the reminder!
Updated!